util/Conventional_Monitor/nwprod/conmon_shared/sorc/conmon_grads_lev.fd/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ cmake_minimum_required(VERSION 2.6)
 
   include_directories( ${UTIL_INC} ${CORE_INCS} ${NCDIAG_INCS} ${NETCDF_INCLUDES} ${MPI_Fortran_INCLUDE_PATH} )
 
-  target_link_libraries( conmon_grads_lev.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${HDF5_Fortran_HL_LIBRARIES} ${NETCDF_LIBRARIES} ${HDF5_Fortran_HL_LIBRARIES} ${HDF5_LIBRARIES} ${MPI_Fortran_LIBRARIES} )
+  target_link_libraries( conmon_grads_lev.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${NETCDF_LIBRARIES} ${MPI_Fortran_LIBRARIES})
```

Comment:
and again.

Reply:
Is this really an MPI program?

Reply:
Without `${MPI_Fortran_LIBRARIES}` link step fails with numerous `undefined reference to MPI*` messages.   The build loads hpc/1.1.0 netcdf/4.7.4.   This netcdf include mpi references.    Therefore, retain MPI_Fortran_LIBRARIES for the time being.

Replace `${NETCDF_LIBRARIES}` with `${NETCDF_Fortran_LIBRARIES}     ${NETCDF_C_LIBRARIES}` in sorc level `CMakeLists.txt`